testing/integration/logs_ingestion_test.go

```diff
@@ -253,8 +253,9 @@ func testFlattenedDatastreamFleetPolicy(
 	policy kibana.PolicyResponse,
 ) {
 	dsType := "logs"
-	dsNamespace := cleanString(fmt.Sprintf("%snamespace%d", t.Name(), rand.Uint64()))
-	dsDataset := cleanString(fmt.Sprintf("%s-dataset", t.Name()))
+	id := uuid.New().String()
+	dsNamespace := cleanString(fmt.Sprintf("namespace-%s", id))
```

Comment:
We should be able to just use `info.Namespace` for this, but id is unique so it doesn't matter too much.

https://github.com/elastic/elastic-agent/blob/a7034ef9c4a1eea4b8caaf983c7ed2f2c18468e9/pkg/testing/define/define.go#L53-L57